Makefile

```diff
@@ -28,17 +28,19 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # This variable is used to construct full image tags for bundle and catalog images.
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# authorino.3scale.net/authorino-operator-bundle:$VERSION and authorino.3scale.net/authorino-operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= authorino.3scale.net/authorino-operator
+# authorino.kuadrant.io/authorino-operator-bundle:$VERSION and authorino.kuadrant.io/authorino-operator-catalog:$VERSION.
+IMAGE_TAG_BASE ?= authorino.kuadrant.io/authorino-operator
 
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= authorino-operator:latest
```

Comment:
Can we please make `quay.io/3scale/authorino-operator:latest` the default here?

Plus future enhancement: CI to automatically push to quay.io when merged to main branch ([example](https://github.com/Kuadrant/authorino/blob/main/.github/workflows/build-latest.yaml)).